cmd/api-errors.go

```diff
@@ -2089,6 +2096,8 @@ func toAPIErrorCode(ctx context.Context, err error) (apiErr APIErrorCode) {
 		apiErr = ErrInvalidBucketName
 	case dns.ErrBucketConflict:
 		apiErr = ErrBucketAlreadyExists
+	case kms.KMSKeyNotFoundException:
+		apiErr = ErrKMSKeyNotFoundException
 	default:
```

Comment:
why not use `kes.ErrKeyNotFound`  ? why do we need a new internal error

Reply:
to keep messaging consistent with AWS API response -if kes error message of `key does not exist` is used, it will be a bit confusing for example here `aws --endpoint-url http://localhost:9001 --profile minio s3api put-object --ssekms-key-id kxxx --bucket test --key issue --body /etc/issue --server-side-encryption aws:kms` whether the error message refers to object vs kms key.